Dockerfile.compile

```diff
@@ -1,8 +1,8 @@
-FROM ubuntu:bionic
+FROM ubuntu:jammy
 RUN apt-get update && apt-get install -y \
-    openjdk-11-jdk \
+    openjdk-17-jdk \
  && rm -rf /var/lib/apt/lists/*
 LABEL maintainer="sig-platform@spinnaker.io"
 ENV GRADLE_USER_HOME /workspace/.gradle
 ENV GRADLE_OPTS "-Xmx12g -Xms12g"
-CMD ./gradlew -PenableCrossCompilerPlugin=true --no-daemon clouddriver-web:installDist -x test
+CMD ./gradlew --no-daemon clouddriver-web:installDist -x test
```

Comment:
`-PenableCrossCompilerPlugin` is in other gradle invocation in github actions too...